git-bot/__main__.py

```diff
@@ -1,3 +1,4 @@
+# https://github-bot-tutorial.readthedocs.io/en/latest/gidgethub-for-webhooks.html#say-thanks
 import os
 import aiohttp
 from aiohttp import web
```

Comment:
Here’s a suggested change:

```suggestion
new_code_here()
```